### ISSUE TITLE
Replace salmon color in CNV categorical data in TCGA projects

### ISSUE THREAD

**Issue by mstreit** [Caleydo/caleydo#1585]
Salmon is too close to the orange selections.

**Comment by cpartl** [comment]
The reason for the salmon color is this diverging color scheme (RdBu) from colorbrewer for 5 categories that we currently use. 
What we can do:
- Choose a different color scheme
- Change the salmon color in that color scheme for 5 categories by hand (I don't recommend this)
- Do not try to use the optimal number of colors from the color schemes for the given number of categories (e.g.. use the 5 defined colors for 5 categories etc.): Use only 3 colors (low, neutral, high) for diverging instead and interpolate between them (this would bring back the "old colors" for cnv)

Any other suggestions?

**Comment by mstreit** [comment]
I think option 1 is most reasonable. I vote for PiYG or PRGn. Both do not have light orange or orange included.

**Comment by ngehlenborg** [comment]
Hm, I'd like to stick to the red-white-blue because that is standard for copy number across TCGA and many other projects. Could we use colors from the gradient that we are using for the heatmaps?

Alternatively, why not change the selection color to something like the purple/pink in the PiYG palette:

![screen shot 2013-08-28 at 4 38 31 pm](https://f.cloud.github.com/assets/1216518/1045248/e96b8668-1021-11e3-97c1-d18bbfcc8185.png)

**Comment by alexsb** [comment]
Orange as a selection color is well tested, much described in papers/help and I personally like it a lot. I also did like the old colors (aka the color gradients we use for the heat maps) better, as Nils suggested. In that case we also don't need to change the selection color.

**Comment by alexsb** [comment]
I think we should simply offer those as an additional choice independent of the color brewer stuff.

**Comment by mstreit** [comment]
I also do not want to change the selection colors because I like them and we chose dataset colors, for instance, to not interfere with the two shades of orange for selection.

I now replaced the salmon with a light red (I took the interpolated one from the histogram). The rest of the colors are still taken from the ColorBrewer scheme.
Here is the result:

OLD:
![cnv_old](https://f.cloud.github.com/assets/4129755/1048581/10187182-108f-11e3-94c7-cfddf82293d4.png)

NEW:
![image](https://f.cloud.github.com/assets/4129755/1048580/0bdd78f6-108f-11e3-989a-7865e7e4c77d.png)
